src/Controls/PlanarControls.js

```diff
@@ -372,6 +373,7 @@ class PlanarControls extends THREE.EventDispatcher {
         }
         if (onMovement) {
             this.view.dispatchEvent({ type: PLANAR_CONTROL_EVENT.MOVED });
+            this.view.dispatchEvent({ type: VIEW_EVENTS.CAMERA_MOVED });
```

Comment:
this could be in another PR. This event is dispatch in GlobeControls when there is movement